### DISCUSSION TITLE
[Backoffice configurator] Generate data schema from MongoDB Views schema

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#310] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

v11.5.2

### Description

It seems that the automatic generation of data schema is disabled if I select a crud-service endpoint connected to a MongoDB View.
I am wondering if this is a bug or a missing feature.

Thanks.

**Comment by TheRusCompass** [comment, marked as answer]
Hi @Lucascanna!
Thanks for reaching out!

It's actually not possible to generate a CRUD schema from Mongo DB Views, the feature is available only for CRUD collections!
